core/src/tangram.cpp

```diff
@@ -214,9 +219,9 @@ void Map::loadSceneAsync(const char* _scenePath, bool _useScenePosition, MapRead
         nextScene = impl->nextScene;
     }
 
-    runAsyncTask([nextScene, _platformCallback, _cbData, this](){
+    runAsyncTask([nextScene, _platformCallback, _cbData, _sceneUpdates, this](){
```

Comment:
Let's simply copy the data from here, we can't rely on the user reference to stay valid when using it in the async task.